Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -91,6 +91,10 @@ theorem eq_one_iff {N : Subgroup G} [N.Normal] (x : G) : (x : G ⧸ N) = 1 ↔ x
   refine QuotientGroup.eq.trans ?_
   rw [mul_one, Subgroup.inv_mem_iff]
 
+@[to_additive (attr := simp)]
+theorem range_mk' : (QuotientGroup.mk' N).range = ⊤ :=
+  (Subgroup.eq_top_iff' _).2 <| by rintro ⟨x⟩; exact ⟨x, rfl⟩
```

Comment:
I am surprised we did not have this result before! I would suggest a slightly different proof:
```suggestion
  MonoidHom.range_eq_top.mpr (mk'_surjective N)
```
since this feels somewhat more conceptually accurate to me.